surefire-booter/src/test/java/org/apache/maven/surefire/booter/SurefireReflectorTest.java

```diff
@@ -55,7 +55,7 @@ public void testShouldCreateFactoryWithoutException()
         ReporterFactory factory = new ReporterFactory()
```

Comment:
Because this PR has only limited scope of refactoring. This PR is aming for refactoring of the interfaces of the listeners and the reason is the fact that we are not able to associate system output with the Thread of the test without refactoring of these interfaces.